docs/index.rst

```diff
@@ -1,6 +1,8 @@
 Welcome to FireSim's documentation (version "|version|")!
 ===========================================================
 
+.. warning:: The documentation currently only describes a AWS EC2 setup of FireSim.
```

Comment:
We might want to remind ourselves to delete this. I guess it's on the front page.